modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZSplitScreenParagraphBehavior.php

```diff
@@ -79,6 +79,20 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
 
     parent::buildBehaviorForm($paragraph, $form, $form_state);
 
+    $form['az_display_settings']['text_width'] = [
+      '#title' => $this->t('Text Width'),
+      '#type' => 'select',
+      '#default_value' => $config['az_display_settings']['text_width'] ?? 'content_width',
+      '#description' => $this->t('Determines the size of the text area.'),
+      '#options' => [
+        'content_width' => $this->t('Content Width'),
+        'full_width' => $this->t('Full Width'),
+      ],
+      '#weight' => 1,
+    ];
+
+    $form['#after_build'][] = [get_class($this), 'afterBuild'];
```

Comment:
I think Drupal core usually reference static method callbacks like this:

```suggestion
    $form['#after_build'][] = '::afterBuild';
```